src/get-started/wordcount-example.md

```diff
@@ -260,13 +320,20 @@ p.run().waitUntilFinish();
 {% github_sample /apache/beam/blob/master/sdks/python/apache_beam/examples/snippets/snippets.py tag:examples_wordcount_minimal_run
 %}```
 
-Note that the `run` method is asynchronous. For a blocking execution instead, run your pipeline appending the `waitUntilFinish` method.
+Note that the `run` method is asynchronous. For a blocking execution, run your
+pipeline appending the <span class="language-java">`waitUntilFinish`</span>
```

Comment:
call the `waitUntilFinish`[+py] method on the result object returned by the call to `run`

(specific wording as appropriate)

Reply:
done